### DISCUSSION TITLE
Refresh mechanism on multiple Fargate Task

### DISCUSSION THREAD

**Discussion by andreas-trvlk** [awspring/spring-cloud-aws#263] [Q&A]
For context, I have been developing a Spring Boot app that will read some properties from the Parameter Store and refresh them using `/actuator/refresh` endpoint. It was working fine so far. But, then I realized that in the production we will have multiple tasks. Executing the endpoint would definitely trigger only one task because they are regulated by a single Load Balancer.

Is there any out-of-the-box solution that we can configure in order to resolve this multiple tasks problem?

**Comment by maciejwalkowiak** [comment]
It's a common issue with `/actuator/refresh` not really specific to Spring Cloud AWS. Take a look here: https://stackoverflow.com/questions/47967899/apply-refresh-on-multiple-instances-annotated-with-refreshscope
